server/models/dataset_helpers/image_metadata.py

```diff
@@ -858,6 +858,20 @@ def _checkMetadataWarnings(clinical):
     return warnings
 
 
+def _extractExifMetadata(data, unstructuredExif):
+    """
+    Add EXIF-related metadata to its own unstructured field.
+
+    :param data: The image metadata.
+    :type data: dict
+    :param unstructuredExif: The dictionary of unstructured EXIF metadata.
+    :type unstructuredExif: dict
+    """
+    for key in list(six.iterkeys(data)):
+        if key.startswith('exif_'):
```

Comment:
Let's do a case-insensitive comparison.

Reply:
The latest revision uses `lower()` in the comparison.